sharktuner/model_tuner/__main__.py

```diff
@@ -4,6 +4,6 @@
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-from . import simple_tuner
+from . import shark_tuner
```

Comment:
Some places use `shark_runer` others `sharktuner`.

Reply:
What do you think about using `sharktuner` uniformly? I don't think the underscore makes python much more readable.